src/Sylius/Behat/Page/Shop/Checkout/SelectPaymentPage.php

```diff
@@ -32,8 +33,16 @@ public function getRouteName()
      */
     public function selectPaymentMethod($paymentMethod)
     {
+        if ($this->getDriver() instanceof Selenium2Driver) {
+            $paymentMethodOption = $this->getElement('payment_method_select');
+            $paymentMethodOption->getParent()->click();
+
+            return;
+        }
+
         $paymentMethodElement = $this->getElement('payment_method');
-        $paymentMethodElement->selectOption($paymentMethodElement->getAttribute('value'));
```

Comment:
You don't use `$paymentMethod` parametr. If someone create a scenario where they select a payment method which is not on first place then this step will always fail. The `payment_method` should find a element with given name an then select it value attribute.